lib/SetAPI/readsalignment/ReadsAlignmentSetInterfaceV1.py

```diff
@@ -2,162 +2,257 @@
 An interface for handling sets of ReadsAlignments.
 """
 
+from SetAPI.error_messages import (
```

Comment:
This is the other class that I looked at in more detail. It's an example of a class that retrieves two different types of Set, even though it can only be used to create one set type. I was looking to standardise it as much as possible as it shares a lot of common features with the `AssemblySetInterfaceV1` class.